### PR TITLE
Add run build action

### PR DESCRIPTION
**Pull request by wpears**
With https://github.com/cfpb/consumerfinance.gov/pull/7732 and https://github.com/cfpb/consumerfinance.gov/pull/7735
Behavior:
- pulls in the frontend build artifact and unzips it
- uses the centos image produced by the builder with the repo files mapped in a volume
- runs create.py inside this container via _build.sh
- pushes the results to S3.

### CODE REVIEW

File: .github/workflows/run-build.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+run the centos image produced by the builder with the repo files mapped in a volume
+runs create.py inside this container
+pushes the results to S3. This thus fully replaces cf.gov-ansible-build
```
Comment (anselmbradford) [inline .github/workflows/run-build.yml:3]: Does yaml allow free text like this? Shouldn't it be preceded by `#` commenting?

File: .github/workflows/run-build.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
```
Comment (anselmbradford) [inline .github/workflows/run-build.yml:18]: Make sense to use `actions/checkout@v3` ?

File: .github/workflows/run-build.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2.27.0
```
Comment (anselmbradford) [inline .github/workflows/run-build.yml:21]: Change to `dawidd6/action-download-artifact@v2`? I think I recall using only major versions was the preferred GHA way.

File: .github/workflows/run-build.yml (hunk 1 above)
Comment (wpears) [reply .github/workflows/run-build.yml:3]: hah! these were just my notes

**Comment by wpears** [comment]
Superseded by https://github.com/cfpb/consumerfinance.gov/pull/7960

**Pull request closed without merging**